elliott/scripts/README.md

```
@@ -0,0 +1,52 @@
+ # Contains helper scripts for JIRA operations
```

markdownlint-cli2 (0.21.0)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)